main.go

```diff
@@ -23,6 +25,32 @@ func requiresGitDir(cmd string) bool {
 var subcommand, subcommandUsage string
 
 func main() {
+        // First thing, set up logging
+        log.SetFlags(log.Ldate|log.Ltime|log.Lshortfile)
+        traceLevel := os.Getenv("DGIT_TRACE")
+        if traceLevel == "" {
+           traceLevel = os.Getenv("GIT_TRACE")
+        } 
+      
+        // If no trace level specified then just dump any log output 
+        if traceLevel == "" {
+           log.SetOutput(ioutil.Discard)
+        } else if traceLevel != "1" && traceLevel != "2" {
```

Comment:
So is the logic here intended to be that if it's 1 or 2 it goes to stderr, otherwise it's assumed that the var is set to a filename to write to?

Reply:
Yes, exactly. I hope that it is reasonable and I believe that it lines up with official git.

Reply:
Okay, I just wanted to make sure I know how to use it. Maybe it should be documented somewhere (the new wiki, maybe?)